dbms/src/Functions/FunctionsGeo.cpp

```diff
@@ -192,5 +278,6 @@ void registerFunctionsGeo(FunctionFactory & factory)
     factory.registerFunction<FunctionPointInPolygon<PointInPolygonFranklin>>();
     factory.registerFunction<FunctionPointInPolygon<PointInPolygonWinding>>();
     factory.registerFunction<FunctionPointInPolygon<PointInPolygonCrossing>>();
+    factory.registerFunction<FunctionPointInPolygon<PointInPolygonWithGrid, true>>();
```

Comment:
Probably we need to leave just one function available to user, under a name `pointInPolygon`.
(I hope, it is the `PointInPolygonWithGrid`).